AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<p>This section feels weak, I feel we can break this section apart, into two.</p><p> </p><p>Arviz is introoduced in the previous section implicity. We could explicitly mention it there.</p><p> </p><p>The inf data stuff could move to the appendix section and I can cover it more in my bayes lesson since I explain what each part is there</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Agreed, done</p>